docs/src/pages/components/cdk-components/sidenav/subtitleapi.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { DxcTable } from "@dxc-technology/halstack-react";
+
+const sidenavSubtitlePropsTable = () => {
+  return (
+    <DxcTable>
+      <tr>
+        <th>Name</th>
+        <th>Default</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>children: node</td>
+        <td></td>
+        <td>The content will be showed as a subtitle in the sidenav.</td>
```

Comment:
Rethink this line, maybe something like: 'Custom content displayed as sidenav subtitle.'.